drivers/atwinc15x0/atwinc15x0_netdev.c

```diff
@@ -42,9 +42,9 @@
 #define ATWINC15X0_MAC_STR          "%02x:%02x:%02x:%02x:%02x:%02x"
 #define ATWINC15X0_MAC_STR_ARG(m)   m[0], m[1], m[2], m[3], m[4], m[5]
 
-#define ATWINC15X0_WAIT_TIME        (1 * US_PER_MS)
+#define ATWINC15X0_WAIT_TIME_MS     (1)
 #define ATWINC15X0_WAIT_TIMEOUT     (20)
-#define ATWINC15X0_WAIT_RECONNECT   (5 * US_PER_SEC)
+#define ATWINC15X0_WAIT_RECONNECT_S (5)
```

Comment:
Why not ms precision here ? This would allow to only have to use ztimer_msec.

Reply:
I thought that `ztimer_sec`isn't a problem at all.

Reply:
Ok, if you only use `ztimer_msec` you need less resources of course.